<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonalPage</class>
 <widget class="QMainWindow" name="PersonalPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1224</width>
    <height>703</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_orario">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLineEdit" name="filename">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="newFile">
     <property name="text">
      <string>New file</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>570</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_orario_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>671</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="2">
        <widget class="QLabel" name="user_email">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lastName">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Last Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>First Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="firstName">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTableWidget" name="tableWidget">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>253</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string>Filename</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Last access</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Collaborators</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QTableWidget" name="tableWidget_2">
       <column>
        <property name="text">
         <string>Filename</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Owner</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Collaborators</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>My last access</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>600</y>
      <width>671</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="placeholderText">
      <string>Put the invitation you received here</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Request to collaborate</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1224</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuChange_Theme">
     <property name="title">
      <string>Change Theme</string>
     </property>
     <addaction name="actionDefault"/>
     <addaction name="actionLight"/>
     <addaction name="actionDark"/>
    </widget>
    <addaction name="actionChange_Password"/>
    <addaction name="menuChange_Theme"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionChange_Password">
   <property name="text">
    <string>Change Password</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
